WcaOnRails/spec/features/eligible_voters_spec.rb

```diff
@@ -20,6 +20,11 @@
       user.team_members.find_by_team_id(wrc_team_id).update!(team_leader: true)
     end
   }
+  let!(:team_senior_member) {
+    FactoryBot.create(:user, :wrc_member).tap do |user|
```

Comment:
Could you add this to the users factory instead? Just like how we do `team_leader` over there. Then you should be able to rewrite this line as:

```ruby
let!(:team_senior_member) { FactoryBot.create(:user, :wrc_member, team_senior_member: true) }
```

Reply:
Sure!

Reply:
I'll also change all the other lines that are written like this.